Misc/NEWS.d/next/Core and Builtins/2023-06-16-21-29-06.gh-issue-105858.Q7h0EV.rst

```diff
@@ -0,0 +1,7 @@
+Improve the constructors for :mod:`ast` nodes. Arguments of list types now
+default to an empty list if omitted. AST nodes now have an
```

Comment:
Perhaps "optional fields are `None` if omitted" is too obvious to mention, but it also seems odd to mention the list case and not the optional case here, since optional fields defaulting to `None` (rather than unset) is also new behavior in this PR.

Reply:
Good point, adding